scripts/requirements/parse.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
```

Comment:
```suggestion
#!/usr/bin/env python3
```

For future-proofing, since unless you have [`python-is-python3`](https://packages.ubuntu.com/search?keywords=python-is-python3) it could be Python 2.x or simply missing.